dashboard/README.md

```diff
@@ -22,6 +22,20 @@ The learn.code.org code is segmented into three parts:
 
 See the main website-ci/README for server install and running instructions.
 
+## Code Style
```

Comment:
How necessary is this file? What's in here that isn't in the README at the root?